pkg/distro/rhel/rhel10/sap.go

```diff
@@ -103,6 +103,11 @@ func sapImageConfig(osVersion string) *distro.ImageConfig {
 			),
 		},
 		// E4S/EUS
+		//
+		// XXX: this must become a declrative option like
```

Comment:
This can/will be fixed once https://github.com/osbuild/images/pull/1464 is merged